susemanager-utils/cluster-providers/caasp/susemanager-cluster-provider-caasp.spec

```diff
@@ -0,0 +1,72 @@
+#
+# spec file for package susemanager-cluster-provider-caasp
+#
+# Copyright (c) 2020 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+%if 0%{?suse_version} > 1320
+# SLE15 builds on Python 3
+%global build_py3   1
+%endif
+
+%define provider caasp
+%define fdir %{_datadir}/susemanager/cluster-providers
+Name:           susemanager-cluster-provider-caasp
+Version:        1.0
+Release:        1
+Summary:        SUSE CaaS Platform cluster provider for SUSE Manager
+License:        GPL-2.0-only
+Group:          Applications/System
+Source:         %{name}-%{version}.tar.gz
+Requires(pre):  coreutils
+Requires:       susemanager-build-keys-web >= 12.0.1
+Requires:       caasp-management-node-formula
+Requires:       caasp-management-settings-formula
+%if 0%{?build_py3}s
+BuildRequires:  python3-pytest
+BuildRequires:  python3-mock
+BuildRequires:  python3-salt
+%else
+BuildRequires:  python-pytest
+BuildRequires:  python-mock
+BuildRequires:  python-salt
+%endif
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildArch:      noarch
+
+%description
+Salt files and metadata for intergrting with SUSE CaaS Platform.
+
+%prep
+%setup -q
+
+%build
+
+%install
+mkdir -p %{buildroot}%{fdir}/states/%{provider}
+mkdir -p %{buildroot}%{fdir}/metadata/%{provider}
+cp -r caasp/* %{buildroot}%{fdir}/states/%{provider}
+cp -r metadata/* %{buildroot}%{fdir}/metadata/%{provider}
+
+%files
+%defattr(-,root,root)
+%dir %{_datadir}/susemanager
```

Comment:
I am pretty sure we have other package that owns this directory. If that's the case, we could have conflicts. We should depend on it. Check a head server.

Reply:
The package won't build without without this `%dir`. I think we're safe (please see my comment above).

Reply:
The usually means a dependency should be added for the package that owns the directory. But as discusses above, there are quite a few, and maybe none of them is correct. So let's keep it.